spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/mapper/UniProtIdTrackerMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.uniprot.store.spark.indexer.uniprot.mapper;
+
+import java.util.Set;
+
+import org.apache.spark.api.java.function.PairFunction;
+import org.uniprot.store.spark.indexer.common.exception.SparkIndexException;
+
+import scala.Tuple2;
+
+public class UniProtIdTrackerMapper implements PairFunction<String, String, Set<String>> {
+
+    @Override
+    public Tuple2<String, Set<String>> call(String line) throws Exception {
+        if (line == null || line.split("\t").length != 2) {
+            throw new SparkIndexException("Unable to parse UniProtIdTrackerMapper line: " + line);
+        }
+        String[] tokens = line.split("\t");
```

Comment:
Do you really want to halt the entire job even if one row is not in the right format?

Reply:
Yes, because it means that the input file is wrong and we need to verify it with SIB